pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -242,11 +242,12 @@ public CompletableFuture<Optional<String>> getChannelOwnerAsync() {
                     // TODO: discard this protocol prefix removal
                     //  by a util func that returns lookupServiceAddress(serviceUrl)
                     if (leader.isPresent()) {
-                        String broker = leader.get().getServiceUrl();
-                        broker = broker.substring(broker.lastIndexOf('/') + 1);
-                        return Optional.of(broker);
+                        return Optional.of(leader.get().getServiceUrl());
```

Comment:
Let's change it back, I find the `ServiceUnitStateChannelTest` will fail after this change because the `PulsarService` started a `LeaderElectionService`